### PR TITLE
add google maps for visited locations

### PR DESCRIPTION
**Pull request by jessinowski**

### CODE REVIEW

File: frontend/src/components/location/LocationMap.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            case "UA_HH_WEST":
+                return {lat: 53.59945171865612, lng: 9.916447204130174};
+            case "UA_NDS":
+                return {lat: 53.66227448816059, lng: 9.979915498076625};
+            case "UA_ST_PAULI":
+                return {lat: 53.55679796532782, lng: 9.970252332632851};
+        }
+    }
+
+    console.log(props.counter);
```
Comment (MarcellNF) [inline frontend/src/components/location/LocationMap.tsx]: no console logs in production code

**Review by MarcellNF — CHANGES_REQUESTED** [review]
One little thing :)

File: frontend/src/components/location/LocationMap.tsx (hunk 1 above)
Comment (jessinowski) [reply frontend/src/components/location/LocationMap.tsx]: yes, I forgot about it :D

**Review by MarcellNF — APPROVED** [review]
LGTM!:)

**Pull request merged**